djangocms_alias/cms_wizards.py

```diff
@@ -13,13 +13,19 @@ class CreateAliasWizard(Wizard):
     def user_has_add_permission(self, user, **kwargs):
         return Alias.can_create_alias(user)
 
+    def get_success_url(self, obj, **kwargs):
+        return obj.get_admin_change_url()
+
```

Comment:
```suggestion
    def get_success_url(self, obj, **kwargs):
        content = obj.get_content()
        return super().get_success_url(self, content, kwargs)
```

The `Alias` situation is different. While the wizard creates an Alias object, the frontend-editable model is `AliasContent`. Therefore turn the `Alias` object into the relevant content object first.

Reply:
@fsbraun,
I tested this feature by installing this PR to djangocms-quickstart locally and creating new alias from wizard button, and it seems that it's working correctly.
I don't know what do you mean here `go to the edit endpoint of Alias.`

Reply:
If you create a page with the wizard, you end up in the edit mode for that page. I would expect the same to happen for the alias: end in edit mode (where you can add plugins) and not in the "settings", i.e. the aliases admin. That would imply a different UX.